wire-ios-data-model/Tests/Source/Model/UserClient/UserClientTests.swift

```diff
@@ -284,15 +354,59 @@ final class UserClientTests: ZMBaseManagedObjectTest {
     }
 
     func testThatItRefetchesMissingFingerprintForUserWithSession() {
-        // given
+        // Given
+        var flag = DeveloperFlag.proteusViaCoreCrypto
+        flag.isOn = true
```

Comment:
maybe we should add an helper method to make sure we always flip back the switch like:
```
extension DeveloperFlag {
    mutating func toggle(_ enabled: Bool, block: () -> Void) {
        self.isOn = enabled
        block()
        self.isOn = !enabled
    }
}
```

Reply:
I was thinking about a change to the `DeveloperFlag` storage so we don't actually use `UserDefaults` in test but just have some in memory storage that we reset in test setup methods. In any case, I think we can do these kind of improvements separately.